harness/chaos/faults/isolate_client_from_cluster.py

```diff
@@ -0,0 +1,37 @@
+from time import sleep
+from sh import ssh
+import logging
+from chaos.faults.types import FaultType
+
+logger = logging.getLogger("chaos")
+
+class IsolateClientFromCluster:
+    def __init__(self, fault_config):
+        self.fault_type = FaultType.RECOVERABLE
+        self.name = "isolate client from cluster"
+        self.redpanda_nodes = None
+        self.client = None
+        self.fault_config = fault_config
+    
+    def inject(self, scenario):
+        self.client = scenario.workload_cluster.nodes[0]
+
+        scenario.workload_cluster.pause_before_send(self.client)
+
+        # Wait to be sure that client stop before send
+        sleep(3)
+
+        scenario.workload_cluster.pause_before_abort(self.client)
+
+        self.redpanda_nodes = []
+        for node in scenario.redpanda_cluster.nodes:
+            self.redpanda_nodes.append(node.ip)
+
+        ssh("ubuntu@"+self.client.ip, "/mnt/vectorized/control/network.isolate.sh", *self.redpanda_nodes)
+        scenario.workload_cluster.resume_before_send(self.client)
+
+        sleep(self.fault_config["sleep_time"])
```

Comment:
We don't need sleep at the end of RECOVERABLE step, by default we already sleep for one minute, if we want to change it for other value it's possible to do via settings